.github/actions/dependencies/action.yml

```diff
@@ -14,7 +14,7 @@ runs:
     - name: install dependencies
       shell: bash
       run: |
-        mkdir ${build_dir}
+        mkdir -p ${build_dir}
```

Comment:
just so that I understand correctly: the `-p` parameter affects only paths which have a depth of at least level i.e. `a/b/...` where there is at least one parent folder.

Reply:
`-p` will create every missing directory on the given path, not just the last name in the path.

Reply:
yeah, that's what I meant. I forgot to type "at least level 1" :P 👍

Reply:
Also, it doesn't error out if the directory already exists. Works around two potential problems with one parameter.

Reply:
okay 👍